backend/capellacollab/projects/crud.py

```diff
@@ -28,10 +28,14 @@ def get_all_projects(db: Session) -> t.List[DatabaseProject]:
     return db.query(DatabaseProject).all()
 
 
-def update_description(
-    db: Session, project: DatabaseProject, description: str
+def update_project(
+    db: Session, project: DatabaseProject, patch_project: PatchProject
 ) -> DatabaseProject:
-    project.description = description
+    if patch_project.name:
+        project.name = patch_project.name
+        project.slug = slugify(patch_project.name)
```

Comment:
Do we want to update the slug when the name changes? Shouldn't slugs be constant (system) identifiers?

Reply:
It's like renaming a project on Gitlab - they have two options: Updating the name and the slug (path). For now, let's just leave it as it is. In the future, we can consider adding a separate route. 
Project renaming is rare anyway.